public/components/trace_analytics/components/common/helper_functions.tsx

```diff
@@ -677,3 +675,83 @@ export const parseHits = (payloadData: string): ParsedHit[] => {
     return [];
   }
 };
+
+export const InjectElementsIntoGrid = (
+  rowCount: number,
+  maxDisplayRows: number,
+  tracesTableMode: string,
+  loadMoreHandler?: () => void
+) => {
+  useEffect(() => {
+    setTimeout(() => {
+      const toolbar = document.querySelector('.euiDataGrid__controls') as HTMLElement;
```

Comment:
nit. it's better to avoid `as` assertions
```suggestion
      const toolbar = document.querySelector<HTMLElement>('.euiDataGrid__controls');
```

Reply:
Removed all as assertions.